src/System.IO.StreamSourceGeneration/src/GenerateStreamBoilerplateAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace System.IO.StreamSourceGeneration;
+
+/// <summary>
+/// Instructs the System.IO.StreamSourceGeneration source generator to generate boilerplate implementations for multiple legacy members, leaving the implementer only with the task of providing implementations for the core operations.
```

Comment:
Even with a widescreen monitor, maximized browser window, and GitHub using a wider view, this almost requires a horizontal scrollbar.

225 characters is way too long.  120 is a much more reasonable length.

The type marker for System.IO.StreamSourceGeneration should be either in `<c>` if it can't be referenced; or as `<see cref="System.IO.StreamSourceGeneration" />` if it can.


```suggestion
/// Instructs the <c>System.IO.StreamSourceGeneration</c> source generator to generate boilerplate implementations for
/// multiple legacy members, leaving the implementer only with the task of providing implementations for the core operations.
```